source/opt/scalar_replacement_pass.cpp

```diff
@@ -898,6 +903,12 @@ bool ScalarReplacementPass::CheckStore(const Instruction* inst,
     return false;
   return true;
 }
+
+bool ScalarReplacementPass::CheckDebugDeclare(uint32_t index) const {
+  if (index != 5u) return false;
```

Comment:
It is confusing. Could you please replace `5u` with a constant variable?

Reply:
OK.